.github/workflows/release.yml

```
@@ -192,15 +209,15 @@ jobs:
 
       - name: Build docker image
         run: |
-          docker build -t gluwa/creditcoin3:${{ env.TAG_NAME }} .
+          docker build -t gluwa/creditcoin3:${{needs.sanity-check.outputs.TAG_NAME  }} --build-arg="BUILD_ARGS=${{ needs.setup.outputs.build_options }}".
```

Results in:

```
Run docker build -t gluwa/creditcoin3:xx.yy.zz-devnet --build-arg="BUILD_ARGS=--features devnet, fast-runtime".
ERROR: "docker buildx build" requires exactly 1 argument.
See 'docker buildx build --help'.
```

see https://github.com/gluwa/creditcoin3/actions/runs/8063126555/job/22024334310


Try:

```suggestion
docker build -t gluwa/creditcoin3:${{needs.sanity-check.outputs.TAG_NAME  }} --build-arg BUILD_ARGS="${{ needs.setup.outputs.build_options }}" .
```

---

that is the most embarrassing one of all